CombinatorialGames/Surreal/Ordinal.lean

```diff
@@ -32,4 +33,34 @@ noncomputable def toSurreal : Ordinal ↪o Surreal where
   inj' _ _ h := toPGame_equiv_iff.1 (Quotient.exact h :)
   map_rel_iff' := @toPGame_le_iff
 
+@[simp]
+theorem _root_.Surreal.mk_toPGame (o : Ordinal) : .mk _ (numeric_toPGame o) = o.toSurreal :=
+  rfl
+
+@[simp]
+theorem toSurreal_zero : toSurreal 0 = 0 :=
+  Surreal.mk_eq toPGame_zero
+
+@[simp]
+theorem toSurreal_one : toSurreal 1 = 1 :=
+  Surreal.mk_eq toPGame_one
+
+theorem toSurreal_injective : Function.Injective toSurreal :=
+  toSurreal.injective
+
+theorem toSurreal_le_iff {a b : Ordinal} : a.toSurreal ≤ b.toSurreal ↔ a ≤ b :=
+  toPGame_le_iff
+
+theorem toSurreal_lt_iff {a b : Ordinal} : a.toSurreal < b.toSurreal ↔ a < b :=
+  toPGame_lt_iff
+
+theorem toSurreal_inj {a b : Ordinal} : a.toSurreal = b.toSurreal ↔ a = b :=
```

Comment:
Is there a reason why this, `toSurreal_lt_iff`, and `toSurreal_le_iff` aren't simp lemmas?

Reply:
Simp already knows about them, I can change the proofs to `by simp` to make that clearer.